build.gradle.kts

```diff
@@ -42,12 +45,42 @@ apply(plugin = "com.xebialabs.dependency")
 group = "ai.digital.deploy.helm"
 project.defaultTasks = listOf("build")
 
+val helmVersion = "3.12.3"
+val operatorSdkVersion = "1.25.4"
```

Comment:
Would it be useful if we have both these variables in gradle.properties. And use them in the build script directly. so helm or operator upgrade will not modify the build script in that case.

If we still require defaults in addtion to the defined project variables in gradle.properties
we can do like `val helmVersionProp = findProperty('helmVersion') ?: 'defaultValue'`